client/my-sites/stats/stats-module/all-time-nav.jsx

```diff
@@ -62,12 +65,21 @@ export const StatsModuleSummaryLinks = ( props ) => {
 		recordStats( item );
 	};
 
+	// Template for standard range options (7, 30, Quarter, Year, All Time).
 	const summaryPath = `/stats/day/${ path }/${ siteSlug }?startDate=${ moment().format(
```

Comment:
Not related to the PR, but again we should always use `momentSiteZone` for today until any further decision is made. cc @Automattic/red